inst/include/generated/model_input.hpp

```diff
@@ -132,7 +132,10 @@ leapfrog::Parameters<ModelVariant, real_type> setup_model_params(const Rcpp::Lis
     const leapfrog::TensorMap1<real_type> hc_age_coarse = parse_data<real_type>(data, "hc_age_coarse", children.hcAG_end);
     const leapfrog::TensorMap2<real_type> abortion = parse_data<real_type>(data, "abortion", children.hAB_ind, proj_years);
     const leapfrog::TensorMap1<real_type> patients_reallocated = parse_data<real_type>(data, "patients_reallocated", proj_years);
-    const leapfrog::TensorMap1<real_type> hc_age_coarse_cd4 = parse_data<real_type>(data, "hc_age_coarse_cd4", 15);
+    const leapfrog::TensorMap1<int> hc_age_coarse_cd4 = parse_data<int>(data, "hc_age_coarse_cd4", options.p_idx_hiv_first_adult);
+    const leapfrog::TensorMap2<real_type> adult_female_inf = parse_data<real_type>(data, "inc", options.p_fertility_age_groups, proj_years);
```

Comment:
Maybe `_inf` to `_infections` here?

Also I see `"inc"` used here; needs updating here if also changed in comment above.

Reply:
Done.